mne/viz/utils.py

```diff
@@ -1184,19 +1205,27 @@ def plot_sensors(info, kind='topomap', ch_type=None, title=None,
                 if pick in value:
                     colors[pick_idx] = color_vals[ind]
                     break
-    if kind == 'topomap':
+    if kind in ('topomap', 'select'):
         pos = _auto_topomap_coords(info, picks, True)
 
     title = 'Sensor positions (%s)' % ch_type if title is None else title
     fig = _plot_sensors(pos, colors, bads, ch_names, title, show_names, axes,
-                        show)
+                        show, kind == 'select')
```

Comment:
Actually it doesn't seem like this should need to block. It could be a kwarg, block=True | False | 'auto', where 'auto'->what you do now. The advantage to doing `fig, sel = plot_sensors(kind='select', block=False)` is that the returned list should get updated in realtime. So if I select something interactively, and then do `print(sel)` I'll see that list, if I select something different, then do `print(sel)` it should be updated with the new selection, etc.